client/pages/events/index.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
+import { EventViewToggle } from '../../components/events'
 import { useAppContext } from '../../context'
-import { EventList } from '../../components/events'
 
 export default function Events() {
   const { events } = useAppContext()
-
-  return <EventList events={events} />
+  const filteredEvents = [];
+  events.forEach(event => filteredEvents.push(event))
+  return <EventViewToggle filteredEvents={filteredEvents}/>
```

Comment:
Can't you just give `EventViewToggle` `events` directly?

```
export default function Events() {
  const { events } = useAppContext()
  return <EventViewToggle filteredEvents={events} />
}
```

Reply:
I'm guessing it's filler code for what @lesliexin is working on. We eventually need to pass in a filter condition. Maybe this filler code is more clear?

```suggestion
  const condition = (event) => true;
  const filteredEvents = events.filter(condition)
  return <EventViewToggle filteredEvents={filteredEvents}/>
```

Reply:
Unfortunately, events is a Map so that filter syntax won't work (needed to convert the map to an array for the child components to work).

Reply:
I think `Array.from(events.values())` might accomplish that